naviswarm/ScanObs[] scanObsBatch
naviswarm/GoalObs[] goalObsBatch
naviswarm/ActionObs[] actionObsBatch
naviswarm/VelocityObs[] velObsBatch
